abr_control/controllers/osc.py

```diff
@@ -138,6 +151,15 @@ def generate(self, q, dq,
             dJ = dJ[:3]
             u_task -= np.dot(dJ, dq)
 
+        if self.ee_adapt != None:
```

Comment:
this is so specific to a single task, there has to be a better way to do this ...

Reply:
We have to have it added in during the control signal generation if we want to do task space learning. When we tried adding it outside it didn't work.

Reply:
yeah, since it's just for a single task though my inclination is to have it in a branch and not in master ...

Reply:
I don't see the harm in having the one if statement? Otherwise we'll need to keep another branch up to date with any changes.

Reply:
broken window theory